doc/nrf/create_application.rst

```diff
@@ -0,0 +1,236 @@
+It specifies the |NCS| version through the :file:`west.yml` `west manifest file`_, which is located in the application :file:`<app>` directory.
+
+With this kind of application, the workspace has the following structure:
+
+.. code-block:: none
+
+   <home>/
+   └─── <west-workspace>/
+      ├─── .west/
+      ├─── nrf/
+      ├─── zephyr/
+      ├─── ...
+      └─── <app>/
+         ├── src/
+         ├── ...
+         └── west.yml
+
+This application type is suitable for the following development cases:
+
+* You want to take advantage of west to manage your own set of repositories.
+* You want to make changes to one or more of the repositories of the |NCS| when working on the application.
+* You want to develop a project that involves more than one build target, for example using a mesh networking protocol like :ref:`ug_matter` or :ref:`ug_bt_mesh`.
+* You want to run a big project that lets you develop most features without having to patch the |NCS| tree, for example with out-of-tree boards, drivers, SoCs, and so on.
+
+For more information about applications placed in workspace in the |NCS|, see the :ref:`workflow 4 on the development model page <dm_workflow_4>`.
+
+.. _create_application_types_freestanding:
+
+Freestanding application
+========================
+
+This kind of application is handled separately from the |NCS|.
+It is located out-of-tree, that is outside of a west workspace, and is not using the `west manifest file`_ to specify the SDK version.
+Instead, the |NCS| version is taken from the :makevar:`ZEPHYR_BASE` environment variable.
+This means that all freestanding applications will use the same |NCS| version and the same copy of the SDK.
+
+With this kind of application, the workspace has the following structure:
+
+.. code-block:: none
+
+   <home>/
+   ├─── <west-workspace>/
+   │  ├─── .west/
+   │  ├─── nrf/
+   │  ├─── zephyr/
+   │  └─── ...
+   └─── <app>/
+      ├─── src/
+      └─── ...
+
+This application type is suitable for the following development cases:
+
+* You prefer to use one copy of the |NCS| when working on one or more applications because of limited bandwidth.
+* You want to do quick prototyping and the results might be later deleted or migrated to an :ref:`application in a workspace <create_application_types_workspace>`.
+
+For more information about freestanding applications in the |NCS|, see the :ref:`workflow 2 on the development model page <dm_workflow_2>`.
+
+Creating an |NCS| application
+*****************************
+
+The process for creating an |NCS| application depends on the development environment.
+Using the |nRFVSC| is the recommended method.
+
+.. _creating_vsc:
+
+Creating application in the |nRFVSC|
+====================================
+
+Use the following steps depending on the application placement:
+
+.. tabs::
+
+   .. group-tab:: Workspace application (recommended)
+
+      To create a workspace application in the |nRFVSC|:
```

Comment:
As this document will be frozen in time, wouldn't it be better to just refer to the VSC documentation here?
This mechanism is correct today, but we will be supporting the next version of NCS for a couple of years, and it's pretty likely that some changes will be made to the west workspace management mechanism.

Reply:
I agree, but in a mid-term. For details, see my answer on Teams.